src/App.js

```diff
@@ -45,16 +72,58 @@ function App() {
       <p>The game to bet on your coin's next transaction</p>
       <br />
       {/* passing setLivePrice as a prop to our BitcoinPrice component */}
-      <BitcoinPrice setCurrentPrice={setCurrentPrice} />
+      <BitcoinPrice
+        setCurrentPrice={setCurrentPrice}
+        referencePrice={referencePrice}
+      />
       <br />
       <div>
-        <button onClick={selectReference}>set reference</button>
-        {/*  updating our state deppending on button pressed*/}
-        <button onClick={() => setBetType(UP_TYPE)}>will go up</button>
-        <button onClick={() => setBetType(DOWN_TYPE)}>will go down</button>
+        {referencePrice ? (
+          <>
+            <p>Your reference price was set at {referencePrice}</p>
+            <p>Now bet to see if the price is going up or down...</p>
+            <br />
+            <button
+              disabled={betType === DOWN_TYPE}
+              onClick={() => setBetType(UP_TYPE)}
+            >
+              🔺
+            </button>
+            <button
+              disabled={betType === UP_TYPE}
+              onClick={() => setBetType(DOWN_TYPE)}
+            >
+              🔻
+            </button>
+          </>
+        ) : (
+          <>
+            <p>Start by selecting a reference price</p>
+            <br />
+            <button onClick={selectReference}>set reference</button>
+          </>
+        )}
       </div>
-      <button onClick={checkPrice}>check!</button>
+      <button disabled={!betType} onClick={checkPrice}>
+        check!
+      </button>
       <br />
+      {showResults ? (
```

Comment:
a fun trick you can do if you don't want to render anything is:

`{showResults && (whatever you want here)}`

that way you can avoid ternary logic if you don't want to return nothing

Reply:
hell yeah thanks for doing it